qiime2/core/tests/test_util.py

```diff
@@ -198,6 +201,31 @@ def test_single_file(self):
                 ('foobarbaz.txt', '93b048d0202e4b06b658f3aef1e764d3')
             ]))
 
+    def test_single_file_md5sum_python(self):
+        with tempfile.TemporaryDirectory() as tempdir:
```

Comment:
Thanks for the suggestion @fabianegli! Since we're getting close to release time I think we shouldn't hold this up for this change, though it may be useful to consider this as a recommendation generally. Would you want to create an issue for this including info on the benefits it would offer?

Reply:
No need to hold off anything. It was just meant as a comment for anyone who wants to go ahead and use pytest fixtures. The benefit is mostly less lines of code (to write and read) and less indentation. Also, if necessary, the root of the temporary directories and files for all tests can be configured in `pytest.ini`.